app/services/slack_service.rb

```diff
@@ -3,6 +3,7 @@ module SlackService
 
   def post_hangout_notification(hangout)
     return unless Features.slack.notifications.enabled
+    return if hangout.hangout_url.blank?
```

Comment:
the idea is to avoid breaking the law of demeter:

https://en.wikipedia.org/wiki/Law_of_Demeter

relating to not reaching into other objects, but just asking them directly.

If `hangout.hangout_url?` is available that neatly sidesteps the issue and is a double win :-)